### PR TITLE
DOCS-3156 Review Notices Articles and remove content that no longer applies to 13.0

### PR DESCRIPTION
**Pull request by tonyriv3**
DOCS-3156 Review Notices Articles and remove content that no longer applies to 13.0

### CODE REVIEW

**Comment by tonyriv3** [comment]
@Mrt134

**Review by Mrt134 — COMMENTED** [review]
I like where this is going, have one comment and see one issue:
- Comment: does this need to merge to master or the 13.0 branch?
- Issue: The drop-downs get hidden inside the tabs box on the new archive page: 
image. Might need to look into either stretching the tabs box vertically to allow the drop-downs to show or change the z-index value so the drop-downs can display on top of the tab box.

**Comment by tonyriv3** [comment]
Perhaps we dont need the drop-downs at all, since the rest of the site doesn't use them. 
This does need to go into the 13.0 branch, but for the sake of building, I have it by itself for now.

**Comment by Mrt134** [comment]
The expandables are a good option too - we'll need to clean up the links inside them though: 
image

**Comment by tonyriv3** [comment]
Already on it!

File: content/Archive.md [hunk 1]
```diff
@@ -8,48 +8,84 @@ Welcome to the Documentation Archive!
 All documentation provided here is End of Life (EoL) and no longer receives **any** updates.
```
Comment (micjohnson777) [inline content/Archive.md:8]:
```suggestion for end-of-life it is not a proper noun so no caps for it, and using hyphens is grammatically correct in this usage
All documentation provided here is end-of-life (EoL) and no longer receives *any* updates.
```

File: content/CORE/System/Update/_index.md [hunk 2]
```diff
@@ -123,7 +123,7 @@ Be aware of these caveats before attempting a major version upgrade:
   Unless new ZFS feature flags are needed, it is safe to leave the pool at the current version.
   If the pool is upgraded, it will not be possible to boot into a previous TrueNAS version that does not support the newer feature flags.
 * Upgrading the firmware of Broadcom SAS HBAs to the latest version is recommended.
```
Comment (micjohnson777) [inline content/CORE/System/Update/_index.md:125]: Not sure how best to fix this list's parallel construction issues other than moving this point to the end and not have it part of the bullet list or create a second bullet list with line 127 which would work.

**Review by micjohnson777 — COMMENTED** [review]
The bullet list needs a rework to achieve parallel construction. Possible to split into two separate lists to achieve this moving the If upgrading in line 124 of the system/udate/_index.md file to a second list with line 127. That might be the easiest way to resolve this issue. Thoughts?

File: content/CORE/System/Update/_index.md (hunk 2 above)
Comment (tonyriv3) [reply content/CORE/System/Update/_index.md:125]: I say we just remove the bullets and use the bold segments as quasi-headers for each point.

**Comment by tonyriv3** [comment]
@micjohnson777 
Suggestions committed! This PR is ready to merge.

**Review by micjohnson777 — APPROVED** [review]

**Pull request merged**